content/docs/studio/troubleshooting.md

```diff
@@ -243,6 +244,17 @@ check that:
 If you get this error and none of the above applies, please
 [get in touch with us](#support).
 
+## Project does not display live metrics and plots
```

Comment:
Should probably link here from some other doc(s) ?

> I suppose this could apply to most troubleshooting sections... currently only discoverable via search.

Reply:
Yes. We can create a separate issue/PR to improve troubleshooting section and links to it. Let's not make part of this PR.

Reply:
Is this one currently linked from somewhere in the Studio UI?

Reply:
No, it isn't